Formula/lacework-cli.rb

```diff
@@ -2,11 +2,31 @@ class LaceworkCli < Formula
   VERSION = "v0.2.12"
   desc "The Lacework Command Line Interface is a tool that helps you manage the Lacework cloud security platform"
   homepage "https://github.com/lacework/go-sdk/wiki/CLI-Documentation"
-  url "https://github.com/lacework/go-sdk/releases/download/#{VERSION}/lacework-cli-darwin-amd64.zip"
   version "#{VERSION}"
-  sha256 "30eccdd2317b09e3ad74c62e4c3c489dfd78a16e98d2afe359cbd0a4506025c8"
   license "Apache-2.0"
 
+  if OS.mac?
+    url "https://github.com/lacework/go-sdk/releases/download/#{VERSION}/lacework-cli-darwin-amd64.zip"
+    sha256 "30eccdd2317b09e3ad74c62e4c3c489dfd78a16e98d2afe359cbd0a4506025c8"
+  elsif OS.linux?
+    case RbConfig::CONFIG['host_cpu'].downcase
+      when "aarch64"
+        url "https://github.com/lacework/go-sdk/releases/download/#{VERSION}/lacework-cli-linux-arm64.tar.gz"
+        sha256 "1c4fb160b741ebefbf35fedff8f569c494ddfdae52e0f9f454618891f7ed7183"   
+      when /i\d86|x86|i86pc/
+        url "https://github.com/lacework/go-sdk/releases/download/#{VERSION}/lacework-cli-linux-386.tar.gz"
+        sha256 "d00b150c97a7219c042d5db896a0ae7e05bd1f80aa2f4eeeedb2e4bce44c5d8b"
+      when /^arm/
+        url "https://github.com/lacework/go-sdk/releases/download/#{VERSION}/lacework-cli-linux-arm.tar.gz"
+        sha256 "4d3cbc3803a5ff85df0265f67d2e5a0b617b03a690eb38782d0dc9e483491d55"
+      when /amd64|x86_64|x64/
+        url "https://github.com/lacework/go-sdk/releases/download/#{VERSION}//lacework-cli-linux-amd64.tar.gz"
+        sha256 "8251fd65dd24d8d8657a4dbe192a18e3dede39e75910806badd1e3004ae4d712"
+      else
+        print "Lacework CLI formula does not support this Architecture"
```

Comment:
```suggestion
        print "The Lacework CLI formula does not support architecture '#{RbConfig::CONFIG['host_cpu'].downcase}'"
```
That 👆🏽, or save the Rbconfig value in a variable and then use it here.